types/react-native/Libraries/Components/TextInput/TextInput.d.ts

```diff
@@ -579,6 +589,12 @@ export interface TextInputProps
    */
   keyboardType?: KeyboardTypeOptions | undefined;
 
+  /**
+   * enum('decimal', 'email', 'none', 'numeric', 'search', 'tel', 'text', 'url')
```

Comment:
Why do we need this? It's not machine-readable and TypeScript already suggests the possible values. Mainly concerne that we update `InputModeOptions` but not this line.

Reply:
I agree with you. I originally added it to match some other props that had them. I pushed a commit that removes hard-coded enums in comments

There are several other places in this file that hard-codes the options in a comment, should I remove those too?